wsd/FileServer.cpp

```diff
@@ -2387,22 +2388,37 @@ void FileServerRequestHandler::preprocessIntegratorAdminFile(const HTTPRequest&
     Poco::replaceInPlace(adminFile, std::string("%ENABLE_DEBUG%"),
                          std::string(enableDebug ? "true" : "false"));
 
-    std::string brandJS(Poco::format(scriptJS, responseRoot, std::string(BRANDING)));
-    std::string brandFooter;
+    const std::string& theme = urv[BRANDING_THEME];
+
+    static const bool useIntegrationTheme =
+        config.getBool("user_interface.use_integration_theme", true);
+    const bool hasIntegrationTheme =
+        !theme.empty() &&
+        FileUtil::Stat(COOLWSD::FileServerRoot + "/browser/dist/" + theme).exists();
```

Comment:
`theme` here should be a directory, directly in the `dist`?
We should limit it so it doesn't contain any relative path for example before we use it.

Reply:
I think, This would be a risky change, as we also use the same path for the cool frame & setting. We don’t have a theme folder there, either.

Anyway, with the recent update I’ve combined the iframe theme logic for both cool and setting into one function, which makes things easier for the future. :)